ACE/ace/INET_Addr.cpp

```diff
@@ -371,8 +371,11 @@ ACE_INET_Addr::set (u_short port_number,
 #endif /* ACE_HAS_IPV6 && ACE_USES_IPV4_IPV6_MIGRATION */
 
 #ifdef ACE_HAS_IPV6
+
+#ifndef AIX
```

Comment:
OS-specific ifdefs in the middle of code are generally a bad idea. Why is this specific to AIX?